cmd2/cmd2.py

```diff
@@ -529,6 +531,9 @@ def __init__(
         # Add functions decorated to be subcommands
         self._register_subcommands(self)
 
+        self.suggest_similar_command = suggest_similar_command
+        self.default_suggestion_message = "The most similar command is {}"
```

Comment:
When I doubt I tend to fall back on what bash does. How about:
', did you mean: {}'

Reply:
It was my first choice. I changed it because of git:
```
git comit

git: 'comit' is not a git command. See 'git --help'.

The most similar command is
	commit
```
But sure, I can change it.